src/solvers/ChambollePock.jl

```diff
@@ -46,8 +46,51 @@ For more details on the algorithm, see[^BergmannHerzogSilvaLouzeiroTenbrinckVida
     > accepted for publication in Foundations of Computational Mathematics
 """
 function ChambollePock(
-    M::mT,
-    N::nT,
+    M::Manifold,
+    N::Manifold,
+    cost::Function,
+    x::P,
+    ξ::T,
+    m::P,
+    n::Q,
+    prox_F::Function,
+    prox_G_dual::Function,
+    forward_operator::Function,
+    adjoint_linear_operator::Function;
+    kwargs...,
+) where {P,T,Q}
+    x_res = allocate(x)
```

Comment:
These allocated variables are never used later.

Reply:
Ah, thanks for noticing, maybe my concentration was already a little off after changing so many interfaces. I corrected this.